Add rule A5 "Boss abuse".

DeviousNull opened the pull request.

DeviousNull committed:
Add rule A5 "Boss abuse".

StanczakDominik commented on CodeOfConduct.md:
```diff
@@ -48,6 +48,11 @@ This section contains things which are less specific and often less problematic
+### 5. Boss abuse
+1. Boss mode is a tool to facilitate a good experience for everyone. Using boss mode to abuse other players is not acceptable, and may result in removal of boss privileges.
+2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
+3. It is not boss abuse to remove a spectator from an ongoing match, or to remove a player from the pre-game lobby.
```
Given other points of the CoC, especially B2, still apply, this is great :)

StanczakDominik commented on CodeOfConduct.md:
```diff
@@ -48,6 +48,11 @@ This section contains things which are less specific and often less problematic
 
+### 5. Boss abuse
+1. Boss mode is a tool to facilitate a good experience for everyone. Using boss mode to abuse other players is not acceptable, and may result in removal of boss privileges.
+2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
```
```suggestion
2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
    - if you are the boss of a lobby, you can still start a kickban *vote* on a player using `!cv kickban <nickname>` ("coordinator vote") instead of `!kickban <nickname>`.
```

DeviousNull replied:
Small tweak: s/nickname/playername/g, since that terminology is already used elsewhere in the CoC:
```suggestion
2. Examples of boss abuse include: kickbanning rule-abiding players after a match has begun, stopping matches because a player claimed a particular start position, and stopping matches to manipulate the skill rating system.
   - if you are the boss of a lobby, you can still start a kickban *vote* on a player using `!cv kickban <playername>` ("coordinator vote") instead of `!kickban <playername>`.
```

StanczakDominik reviewed (commented):
Actually, how about this?

DeviousNull committed:
Add tip for how to start a kickban vote as a boss

Add a tip under A5.2, the boss-abuse example about kickbanning, mentioning how to call a kickban vote as a boss.
